src/stepper.js

```diff
@@ -1,6 +1,12 @@
+/* @flow */
 const errorMargin = 0.0001;
 
-export default function stepper(frameRate, x, v, destX, k, b) {
+export default function stepper(frameRate: number,
+                                x: number,
```

Comment:
It does pass ESLint checks (you might want to change the ESLint config), but I can change it anyway. Is it just the first argument on a new line?